pom.xml

```diff
@@ -132,6 +132,13 @@
 
 	<dependencyManagement>
 		<dependencies>
+			<!-- force latest reactor-netty -->
```

Comment:
@marcingrzejszczak is it ok to merge pinning this for now as we can't control the upstream versions and break unless very latest?

Reply:
Of course